src/protoc-gen-twirp-ts/plugin.ts

```diff
@@ -60,11 +66,21 @@ export class ProtobuftsPlugin extends PluginBase<File> {
         ctx.symbols.register(createLocalTypeName(descriptor, registry), descriptor, messageFileOut);
       });
 
-      // Twirp generation
-      const twirpFileOut = new File(`${fileDescriptor.name?.replace(".proto", "").toLowerCase()}.twirp.ts`);
-      const twirpContent = await generate(ctx, fileDescriptor);
-      twirpFileOut.setContent(twirpContent);
-      files.push(twirpFileOut);
+      // Twirp server generation
+			if (!params.disable_server_generation) {
+				const twirpFileOut = new File(`${fileDescriptor.name?.replace(".proto", "").toLowerCase()}.twirp.ts`);
+				const twirpContent = await generateServer(ctx, fileDescriptor);
+				twirpFileOut.setContent(twirpContent);
+				files.push(twirpFileOut);
+			}
+
+			// Twirp client generation
+			if (!params.disable_client_generation) {
+				const twirpClientFileOut = new File(`${fileDescriptor.name?.replace(".proto", "").toLowerCase()}.twirp-client.ts`);
```

Comment:
Hey @jeffreylo thanks for the PR,

This is a significant breaking change at generating the client and the server separately. 
Maybe we can approach slightly differently as I'd like to have this in but keeping backwards compatibility.

What we could do is to have the following flags:

- `--twirp_opt=standalone`            // Generates client and server in 2 separate files (applies only if the below options aren't provided)
- `--twirp_opt=client_only`            // Only client will be generated
- `--twirp_opt=server_only`          // Only server will be generated


If none of the above options is provided, the generator will keep client & server in the same file

Reply:
Thanks for the feedback, @fenos — working through this today, I think there’s also more things to consider as the underlying libraries in `twirp-ts` will still rely on node-specific libraries, e.g., the `import * from http` lines. That seems like it will require much more consideration. Any thoughts?

Reply:
I believe that the client side code doesn't need import twirp-ts at all so it should be fine